src/backend/marsha/core/utils/medialive_utils.py

```diff
@@ -308,6 +313,32 @@ def stop_live_channel(channel_id):
     """Stop an existing medialive channel."""
     medialive_client.stop_channel(ChannelId=channel_id)
 
+def send_message_channel(video):
+    channel_id = video.get_medialive_channel().get("id")
```

Comment:
If there is channel configured, you should abort the function by returning `None` for example. While the `start-live` endpoint has not been called, all the AWS information are not existing. So `video.live_info` is `None`